test/storages/sequelize.test.ts

```diff
@@ -0,0 +1,408 @@
+/* eslint-disable @typescript-eslint/promise-function-async */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
+import { SequelizeStorage as Storage } from '../../src/storages/SequelizeStorage';
+
+import * as sequelize from 'sequelize';
+import { join } from 'path';
+import { v4 as uuid } from 'uuid';
+import jetpack = require('fs-jetpack');
+
+describe('sequelize', () => {
+	jetpack.cwd(__dirname).dir('tmp', { empty: true });
+
+	const helper = {} as any;
+	beforeEach(() => {
+		Object.assign(helper, {
+			sequelize: new sequelize.Sequelize('database', 'username', 'password', {
+				dialect: 'sqlite',
+				storage: this.storagePath,
+				logging: false,
+			}),
+			storagePath: join(__dirname, `/../tmp/storage-${uuid()}.sqlite`),
+		});
+	});
+
+	describe('constructor', () => {
+		it('requires a "sequelize" or "model" storage option', () => {
+			// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+			// @ts-ignore
+			expect(() => new Storage()).toThrowError('One of "sequelize" or "model" storage option is required');
+		});
+
+		it('stores needed options', () => {
+			const storage = new Storage({ sequelize: helper.sequelize });
+			expect(storage).toHaveProperty('sequelize');
+			expect(storage).toHaveProperty('model');
+			expect(storage).toHaveProperty('columnName');
+		});
+
+		it('accepts a "sequelize" option and creates a model', () => {
+			const storage = new Storage({ sequelize: helper.sequelize });
+			expect(storage.model).toBe(helper.sequelize.model('SequelizeMeta'));
+			expect(storage.model.getTableName()).toBe('SequelizeMeta');
+			return storage.model
+				.sync()
+				.then(model => model.describe())
+				.then(description => {
+					expect(description).toMatchInlineSnapshot(`
+						Object {
+						  "name": Object {
+						    "allowNull": false,
+						    "defaultValue": undefined,
+						    "primaryKey": true,
+						    "type": "VARCHAR(255)",
+						  },
+						}
+          `);
```

Comment:
Looks like you forgot to fix this last line in your `use tabs` commit (b840f0ec624438e9f1b00fda3dc2f7b35b1dd09f)

Also, by the way, I'm curious - does Jest automatically outdents the inline snapshot?

Reply:
Good catch - and yes jest is supposed to handle this automatically, but I think I need to tweak some of the lint config so it can tell that we like tabs over spaces. I'll do that in another PR I think and just fix this one manually.